diff --git a/lib/plugins/signon-plugins-common/SignOn/blobiohandler.cpp b/lib/plugins/signon-plugins-common/SignOn/blobiohandler.cpp
index bb00e7d..8d2d974 100644
--- a/lib/plugins/signon-plugins-common/SignOn/blobiohandler.cpp
+++ b/lib/plugins/signon-plugins-common/SignOn/blobiohandler.cpp
@@ -25,6 +25,7 @@
 
 #include <QDBusArgument>
 #include <QBuffer>
+#include <QDataStream>
 #include <QDebug>
 
 #include "SignOn/signonplugincommon.h"
@@ -236,4 +237,4 @@ QVector<QByteArray> BlobIOHandler::pageByteArray(const QByteArray &array)
     pagingBuffer.close();
 
     return dataPages;
-}
+}
\ No newline at end of file
